openlibrary/templates/merge_request_table/table_header.html

```diff
@@ -0,0 +1,93 @@
+$def with (total, librarians, mode, username, submitter, reviewer, status)
```

Comment:
```suggestion
$def with (totals, librarians, mode, username, submitter, reviewer, status)
$# totals : dict : {"open": int, "closed": int} : The total number of merge requests found for each mode
$# librarians : dict : {"submitters": list[str], "reviewers": list[str]} : Collection of reviewer and submitter usernames
$# mode : str : "open" | "closed" : Type of requests that are currently being viewed.
$# username : str | None : Username of authenticated patron, or `None` if patron is unauthenticated.
$# submitter : str | None : Submitter's username.  Only passed if we are filtering requests by `submitter`.
$# reviewer : str | None : Reviewer's username. Only passed if we are filtering requests by `reviewer`.
$# status : str | None : "0" | "1" | "2" : Represents a request's status.  Only passed if we are filtering by `status`

```
Add comments about the template's parameters.  Change `total` to `totals`, as this dict holds multiple totals.